### PR TITLE
Add tuple type

### PR DESCRIPTION
**Pull request by Viterkim**
[sc-114872]

### CODE REVIEW

File: src/validators/undefined.ts [hunk 1]
```diff
@@ -95,6 +121,8 @@ export class RequiredUndefined extends UndefinedValidator {
   }
 }
 
+
+  // TODO: shouldn't we just remove this type?
```
Comment (Viterkim) [inline src/validators/undefined.ts]: Note this

File: src/validators/undefined.test.ts [hunk 2]
```diff
@@ -93,12 +99,25 @@ describe.each([false, true])('Undefined (optimize: %s)', optimize => {
     })
   })
 
+  // TODO: shouldn't we just remove this type?
```
Comment (Viterkim) [inline src/validators/undefined.test.ts]: Note this

File: src/validators/object.ts [hunk 3]
```diff
@@ -129,20 +129,52 @@ export abstract class ObjectValidator<T extends ObjectSchema = never, O = never>
...
+  }
+
+  private typeString(options?: ValidatorExportOptions): string {
+    const language = options?.language ?? 'typescript'
+    switch (language) {
+      case 'typescript': {
+        const lines = Object.keys(this.schema).map(k =>
+          `'${k}': ${this.schema[k].toString(options)}`.replace(/(^|\n)/g, '$1  ')
+        )
+        let typeStr = `{\n${lines.join('\n')}\n}`
```
Comment (tlbdk) [inline src/validators/object.ts:150]: To make it really nice you could handle the case where the object is empty {} and don't add the new lines.

fx.

``` typescript
let typeStr =  lines === '' ? `{}` : `{\n${lines.join('\n')}\n}`
```

**Review by tlbdk — APPROVED** [review]

File: src/validators/object.ts (hunk 3 above)
Comment (Viterkim) [reply src/validators/object.ts:150]: good idea

**Pull request merged**